sections/06_discussion.md

```diff
@@ -137,3 +137,7 @@ interpretability problems, how can we best ensure reproducible models? What
 might a clinician, or policy maker, need to see in a deep model in order to
 influence healthcare decisions? Or, is deep learning a hypothesis generation
 machine that requires manual validation?*
+
+### Transfer learning/transferability of features
```

Comment:
I guess I prefer to introduce in specific sections and cross-domain recap here. I think it's going to be key in bio for many near-future advances.